torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -103,25 +103,18 @@ class ShardingStrategy(Enum):
                    GPU memory until backward computation is done. It inserts reduce_scater
                    after backward computation for synchronizing and sharding gradients.
                    Sharded optimizer states are updated locally.
-    SHARD_OP(future support): Shard optimizer states only, this algorithm inserts all_gather
-                              or broadcast before forward computation and keeps the full
-                              parameters in GPU memory until backward computation is done. It
-                              inserts all_reduce after backward computation for synchronizing
-                              gradients. Sharded optimizer states are updated locally.
-    NO_SHARD(future support): This is similar to PyTorch `DistributedDataParallel` API.
-                              Parameters, gradients and optimizer states are replicated
-                              among ranks, all_reduce is inserted after backward computation
-                              is done for synchronizing gradients. Full optimizer states
-                              are updated in each rank.
+    NO_SHARD: This is similar to PyTorch `DistributedDataParallel` API. Parameters, gradients
+              and optimizer states are replicated among ranks, all_reduce is inserted after
```

Comment:
nit: same as on the previous PR. Double quote `DistributedDataParallel` and `all_reduce` to display as code.